src/Routes.js

```diff
@@ -43,13 +43,15 @@ import NHGRIpilotinfo from './pages/NHGRIpilotinfo';
 import { Status } from './pages/Status';
 import { SummaryVotes } from './pages/SummaryVotes';
 import HomeResearcherInfo from "./pages/HomeResearcherInfo";
+import BackgroundSignIn from './pages/BackgroundSignIn';
 
 
 const Routes = (props) => (
   <Switch>
     <Route exact path="/" render={(routeProps) => <Home {...routeProps} {...props} />} />
     <Route exact path="/home" render={(routeProps) => <Home {...routeProps} {...props} />} />
     <Route exact path="/status" render={(routeProps) => Status(_.mergeAll([routeProps, props]))} />
+    <Route exact path="/backgroundsignin" component={BackgroundSignIn} />
```

Comment:
This is more of a comment for discussion, feel free to reply back @rfricke-asymmetrik @rushtong. I don't have any issues with the test exclusive component, but I'd feel a lot better if the Route was only rendered in select environments (say if env="test" or env="local"). 

This way you could pass in the env variable as a prop into Routes and have a line like ```{props.env === 'test' && <Route exact path="/backgroundsignin" component={BackgroundSignIn} />``` to ensure that the Route only shows on a test env.

In terms of getting the env variable, we have a file called Config that is already being imported within App.js that contains a function called getEnv. You could use that function as part of the Application initialization process and pass it down into props from there.

Reply:
I think I'd prefer that we have this option available on any env **_except_** prod. In the (hopefully near) future, we'll have on-demand k8s clusters that might point to a perf, alpha, or staging env and I'd like to be able to have comparison runs across different envs with different builds.

Reply:
Agreed. This should, at the very least, not be visible on prod.